Assets/Scripts/Player/PlayerController.cs

```diff
@@ -31,6 +40,7 @@ void MovePlayer()
         Vector3 movement = new Vector3(moveHorizontal, 0, moveVertical);
 
         // Move the player object
-        transform.Translate(movement * moveSpeed * Time.deltaTime);
+        Vector3 clampedSpeed = Vector3.ClampMagnitude(movement * moveSpeed, maxSpeed);
```

Comment:
We should multiply movement with `Time.deltaTime` independent from frame rate (or use `FixedUpdate()` method to call `MovePlayer()`)